src/components/touchableOpacity/index.tsx

```diff
@@ -62,6 +62,11 @@ type Props = BaseComponentInjectedProps &
 class TouchableOpacity extends PureComponent<Props, {active: boolean}> {
   static displayName = 'TouchableOpacity';
 
+  static defaultProps = {
+    throttleTime: 0,
+    throttleOptions: {leading: true, trailing: false}
+  }
```

Comment:
This will override our config in private uilib.. 
Why did you add it?

Reply:
This is to replace the configuration in ThemeManager (line #16).

Reply:
Ohh I see now..
The thing is that setting it as `defaultProps` give it a different priority. 
Unfortunately, defaultProps are prioritized over theme props and override them. 
And the way it was defined before was considered as theme props..

Another option we can do is to use default Constants like we do sometimes when we extract the values 
WDYT?

Reply:
I bit messy but I guess that could be a solution. I'll push a fix

Reply:
Done